extensions/amp-youtube/1.0/component.js

```diff
@@ -35,7 +35,7 @@ const PlayerStates = {
   '-1': 'unstarted',
   '0': 'ended',
   '1': 'playing',
-  '2': 'paused',
+  '2': 'pause',
```

Comment:
I agree. But these are events. Or at least used as events. And the event's name is [pause](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/pause_event), which is also inconsistent in Web spec, since the next event's name is [ended](https://developer.mozilla.org/en-US/docs/Web/API/HTMLMediaElement/ended_event). But the goal here is to make other video players to dispatch `video`'s events. /cc @alanorozco